src/scss/pages/_contact.scss

```diff
@@ -25,120 +25,119 @@
             height: 214px;
             justify-content: space-around;
             width: 286px;
+        }
 
-            &-box {
-                display: flex;
-            }
+        &-item {
+            display: flex;
+            flex-direction: column;
+        }
 
-            &-item {
-                display: flex;
-                flex-direction: column;
-            }
+        &-box {
+            display: flex;
```

Comment:
Delete this class and use the bootstrap class

Reply:
done